js/mage/adminhtml/input-counter.js

```diff
@@ -0,0 +1,41 @@
+// https://stackoverflow.com/a/44436408/5703627
+document.observe('dom:loaded', function() {
+    Element.addMethods({
+        // setup once, memoize the counter element and maxLen
+        prepare_for_countdown: function(element) {
+            var elm = $(element);
+            // even if you call it multiple times, it only works once
+            if(!elm.retrieve('counter')) {
+                var counter = new Element('span');
+                elm.next('.note').insert(counter);
+                elm.store('counter', counter);
+                var maxLen = elm.className.match(/maximum-length-(\d+)/)[1];
+                elm.store('maxLen', maxLen);
+            }
+            return elm; // so you can chain
+        },
+        // display the value, run once at load and on each observed keyup
+        countdown: function(element) {
+            var elm = $(element);
+            var curLen = elm.getValue().length;
+            var maxLen = elm.retrieve('maxLen');
+            var count  = maxLen - curLen;
+            var counter = elm.retrieve('counter');
+            counter.update(' (' + curLen + '/' + maxLen + ')');
+            if (curLen > maxLen) {
+                counter.setStyle({'color': 'red'});
+            } else {
+                counter.setStyle({'color': 'inherit'});
+            }
+            return elm;
+        }
+    });
+
+    // run setup and call countdown once outside of listener to initialize
+    $$('.validate-length').invoke('prepare_for_countdown').invoke('countdown');
```

Comment:
can we not use prototype?

Reply:
Of course, but I dont want to rewrite my 6y old code ;)

Reply:
I thought it was a couple of lines of prototypejs but actually it's all of it 😅

Reply:
My only argument is "it works" ... and it might be helpful.